src/rippled/lib/rippled.js

```diff
@@ -248,6 +248,7 @@ const getAccountTransactions = (
   account,
   limit = 20,
   marker = '',
+  reverseOrder = false,
```

Comment:
```suggestion
  opts = {
    limit = 20,
    marker = '',
    reverseOrder = false
  }
```
I don't think that's quite legal JS but you get the idea - it'll simplify the number of random `undefined` params we need to add.

Reply:
I see what youre saying. Right now it's just one parameter in one place so I don't think it matters. In the future we can simplify if we find that we're adding more parameters.